relay-config/src/config.rs

```diff
@@ -1336,6 +1336,41 @@ impl Default for Health {
     }
 }
 
+///
+#[derive(Serialize, Deserialize, Debug)]
+#[serde(default)]
+pub struct Cogs {
+    /// Whether COGS measurements are enabled.
+    enabled: bool,
+    /// Granularity of the COGS measurements.
+    ///
+    /// Measurements are aggregated based on the granularity.
+    granularity: Option<Duration>,
```

Comment:
The config usually types time durations as `u32` and provides an accessor to build a `Duration` from it. We should keep it consistent.